File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for num in nums :
            val = nums.index(num)
            nums[val] = -100
            if ((target - num) in nums ) and (val != nums.index(target-num)) :
                return list((val,nums.index(target-num)))
        

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int answer[] = new int[2];
        int check = 0;
        for (int i = check + 1; i < nums.length; i++) {
            if (target - nums[check] == nums[i] && check != i) {
                answer[0] = check;
                answer[1] = i;
                break;
            } else if (i == nums.length - 1) {
                check ++;
                i = 0;
            }
        }
        return answer;
    }
}